### DISCUSSION TITLE
Upgrading from 11.1.0 to 12.0.0

### DISCUSSION THREAD

**Discussion by anheri** [Gremlinq/ExRam.Gremlinq#1438] [Q&A]
After upgrading ExRam.Gremlinq.Core and ExRam.Gremlinq.Providers.CosmosDb.AspNet from 11.1.0 to 12.0.0 and Gremlin.Net from 3.6.1 to 3.7.1
I can not get a response from a gremlinq query. It just waits for minutes without ever returning.
I can get data using the gremlin.net client, so the server setup seems to be good.

I had to move the specification of the partitionKey from the UseCosmosDb to appsettings.json. ( seems like a breaking change in 12.0.0)

`            services.AddGremlinq(setup =>
            {
                setup.UseCosmosDb<Model.Graph.Vertex, Model.Graph.Edge>();               
            });
`

nothing is emitted in the logs... anybody have any idea of what the h"#¤ll is going on?

**Comment by danielcweber** [comment]
A minimal code setup to reproduce this would help everybody.

**Comment by danielcweber** [comment]
No repro no issue no problemo.
